packages/components/organisms/f-cookie-banner-static/package.json

```diff
@@ -8,7 +11,7 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-cookie-banner": "file:../dist"
+    "@justeat/f-cookie-banner": "3.8.4"
```

Comment:
Figured we should probably be pointing at a version, rather than referencing file paths.